aict_tools/preprocessing.py

```diff
@@ -163,8 +163,8 @@ def horizontal_to_camera(df, model_config):
             alt_pointing=alt_pointing,
             focal_length=df[model_config.focal_length_column],
         )
-
-    elif model_config.coordinate_transformation == 'FACT':
+    # simple is probably not a very good name here :/
```

Comment:
Maybe call it `flat`?

Reply:
I dont mind either name.
Removed the coordinate transformation key as I felt its no longer needed and then realized you might also want a simple structure with a non-FACT coordinate transformation.
Opinions on reinserting that key?

Reply:
yes, should be still an option